suite-native/passphrase/src/useHandleDuplicitPassphrase.ts

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { useAlert } from '@suite-native/alerts';
+import { switchDuplicatedDevice } from '@suite-common/wallet-core';
+import {
+    AppTabsRoutes,
+    HomeStackRoutes,
+    PassphraseStackParamList,
+    PassphraseStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import { TrezorDevice } from '@suite-common/suite-types';
+import { useTranslate } from '@suite-native/intl';
+
+import { selectPassphraseError } from './passphraseSlice';
+
+type NavigationProp = StackToStackCompositeNavigationProps<
+    PassphraseStackParamList,
+    PassphraseStackRoutes,
+    RootStackParamList
+>;
+
+export const useHandleDuplicitPassphrase = () => {
```

Comment:
nit:
```suggestion
export const useHandleDuplicatePassphrase = () => {
```

Reply:
I don't agree. Duplicit is the correct word here